src/components/IconButton/IconButton.module.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../styles/theme' as theme;
+
+.IconButton {
+  border-radius: 50%;
+  padding: 2px;
+  line-height: 0px;
+
+  cursor: pointer;
+  background: none;
+  border: none;
+  display: inline-block;
+  margin: 0px;
+
+  &:hover {
+    background-color: theme.$color-greyscale-transparency-3;
+  }
+}
```

Comment:
I didn't do a design review in Storybook but it looks like the `:active` styles from Figma are missing here.

[Icon Button in Figma](https://www.figma.com/file/wTOVNZonl9MmEzORwBHDuq/ZERO-Design-System-(zDS)?node-id=5043%3A106692&t=zdrU2dk3zXSbz2z0-1)

Reply:
Oh, strange, those are different than the one [I was sent](https://www.figma.com/file/wTOVNZonl9MmEzORwBHDuq/ZERO-Design-System-(zDS)?node-id=8718%3A43245&t=iqkZtphiZs1SMdsr-0).

I'll confirm.

Reply:
Ah yes I see - the one you were sent is the `Icon` component, which is different to `IconButton` by the looks of it. `IconButton` is under `IconGroup`, so a little bit hard to find!

Reply:
Yeah, turns out the design group is gonna tweak the designs I was given and I'll follow up with a new PR probably. I'm going to close this one until then.